easybuild/easyconfigs/c/CGAL/CGAL-4.14.3-iccifort-2020.1.217.eb

```diff
@@ -0,0 +1,26 @@
+name = 'CGAL'
+version = '4.14.3'
+
+homepage = 'https://www.cgal.org/'
+description = """The goal of the CGAL Open Source Project is to provide easy access to efficient 
+ and reliable geometric algorithms in the form of a C++ library."""
+
+toolchain = {'name': 'iccifort', 'version': '2020.1.217'}
+toolchainopts = {'strict': True}
+
+source_urls = ['https://github.com/CGAL/cgal/releases/download/releases%2FCGAL-%(version)s']
+sources = [SOURCE_TAR_XZ]
+checksums = ['5bafe7abe8435beca17a1082062d363368ec1e3f0d6581bb0da8b010fb389fe4']
+
+dependencies = [
+    ('Boost', '1.72.0'),
+]
```

Comment:
@bartoldeman where did all the dependencies of the previous version go? they are not needed anymore?